example/C/src/Patterns/Chain_02_Pipeline.lf

```diff
@@ -15,7 +15,7 @@
  * @author Edward A. Lee
  */
 target C {
-    threads: 4,
+    workers: 4,
```

Comment:
I think here it makes sense to leave the parameter so that it shows how the number of workers can be adjusted.